### DISCUSSION TITLE
Missing a lot of Arabic Channels

### DISCUSSION THREAD

**Discussion by ash95sa** [iptv-org/community#613] [Q&A]
We need your support to add the below Arabic channels to the list

Saudi 1
Saudi SBC
Saudi Thikrayat

Kuwait AlQurain
Kuwait AlAdalah
Kuwait AlRai
Kuwait Shahid

Iraq Maaref
Iraq Anwar 1
Iraq AlAqilah
Iraq Saut Alaqilah
Iraq Alawhad
Iraq Kawther

Egypt ON tv
Egypt ON Drama 
Egypt Hayat
Egypt Hayat Drama

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
